### DISCUSSION TITLE
Build on CI return missing files errors

### DISCUSSION THREAD

**Discussion by CezaryCCodigee** [pointfreeco/swift-dependencies#281] [General]
### Description

Hi, our CI (Code Magic) returns some errors. Maybe you guys have idea what happens? Locally build works perfectly.

`Error (Xcode): Build input files cannot be found: '/Users/builder/Library/Developer/Xcode/DerivedData/Runner-edaimyiflreloheqntgnhkmwcclv/Build/Intermediates.noindex/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/Dependencies.o', '/Users/builder/Library/Developer/Xcode/DerivedData/Runner-edaimyiflreloheqntgnhkmwcclv/Build/Intermediates.noindex/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/XCTestDynamicOverlay.o', '/Users/builder/Library/Developer/Xcode/DerivedData/Runner-edaimyiflreloheqntgnhkmwcclv/Build/Intermediates.noindex/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/IssueReporting.o', '/Users/builder/Library/Developer/Xcode/DerivedData/Runner-edaimyiflreloheqntgnhkmwcclv/Build/Intermediates.noindex/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/CombineSchedulers.o', '/Users/builder/Library/Developer/Xcode/DerivedData/Runner-edaimyiflreloheqntgnhkmwcclv/Build/Intermediates.noindex/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/Clocks.o', '/Users/builder/Library/Developer/Xcode/DerivedData/Runner-edaimyiflreloheqntgnhkmwcclv/Build/Intermediates.noindex/ArchiveIntermediates/Runner/BuildProductsPath/Release-iphoneos/ConcurrencyExtras.o'. Did you forget to declare these files as outputs of any script phases or custom build rules which produce them?`

### Checklist

- [ ] I have determined whether this bug is also reproducible in a vanilla SwiftUI project.
- [ ] If possible, I've reproduced the issue using the `main` branch of this package.
- [X] This issue hasn't been addressed in an [existing GitHub issue](https://github.com/pointfreeco/swift-dependencies/issues) or [discussion](https://github.com/pointfreeco/swift-dependencies/discussions).

### Dependencies version information

1.4.1

### Xcode version information

Xcode 16

### Swift Compiler version information

```shell
swift-driver version: 1.115 Apple Swift version 6.0 (swiftlang-6.0.0.9.10 clang-1600.0.26.2)
Target: arm64-apple-macosx15.0
```

**Comment by mbrandonw** [comment]
Hi @CezaryCCodigee, I don't know off hand what could be wrong, but I do not think this is a problem with the library. It is more likely a problem with your CI/build process. So I am going to convert this to a discussion, but if you provide more information there is a chance someone may be able to help you.

**Reply by CezaryCCodigee** [reply]
thanks for quick reply :) to be honesty I have no idea what more I can say :(
